omnibus/config/software/pympler.rb

```diff
@@ -9,34 +9,17 @@
 name 'pympler'
 default_version "0.7"
 
-if with_python_runtime? "3"
-  dependency 'pip3'
-  dependency 'setuptools3'
-end
-
-if with_python_runtime? "2"
-  dependency 'pip2'
-end
+dependency 'pip3'
+dependency 'setuptools3'
```

Comment:
where is `setuptools3` dependency used?

Reply:
It's required by:
* `pympler`
* `datadog-agent-integrations-py3.dependencies` (so `datadog-agent-integrations-py3`)